rootfs/api/serializers.py

```diff
@@ -134,7 +134,7 @@ class ConfigSerializer(serializers.ModelSerializer):
     owner = serializers.ReadOnlyField(source='owner.username')
     values = JSONFieldSerializer(required=False, binary=True)
     memory = JSONFieldSerializer(required=False, binary=True)
-    cpu = JSONFieldSerializer(required=False, binary=True, type='int')
+    cpu = JSONFieldSerializer(required=False, binary=True)
```

Comment:
since this is the only place that needed `int` can you also remove that from the `JSONFieldSerializer`, unless you want float specifically then you could change that maybe? no big deal either way